moderner-cv/recipe.yaml

```diff
@@ -13,12 +13,12 @@ source:
     - 0001-Change-imports-from-preview-to-typst-forge.patch
 
 build:
-  number: 2
+  number: 3
   noarch: generic
   script:
     content:
       - mkdir -p "$PREFIX/share/typst/packages/typst-forge/${{ name }}/${{ version }}"
-      - cp -r typst.toml README.md lib.typ template/
+      - cp -r typst.toml README.md lib.typ template
```

Comment:
on my macOS machine, it failed to build otherwise

```
 │ │ Files in package:
 │ │   - share/typst/packages/typst-forge/moderner-cv/0.1.0/README.md
 │ │   - share/typst/packages/typst-forge/moderner-cv/0.1.0/example.typ
 │ │   - share/typst/packages/typst-forge/moderner-cv/0.1.0/lib.typ
 │ │   - share/typst/packages/typst-forge/moderner-cv/0.1.0/typst.toml
 │ │   - info/about.json
 │ │   - info/hash_input.json
 │ │   - info/index.json
 │ │   - info/licenses/LICENSE
 │ │   - info/paths.json
 │ │   - info/recipe/0001-Change-imports-from-preview-to-typst-forge.patch
 │ │   - info/recipe/recipe.yaml
 │ │   - info/recipe/rendered_recipe.yaml
 │ │   - info/recipe/variant_config.yaml
 │ │   - info/tests/tests.yaml
```